src/SldStyleParser.ts

```diff
@@ -95,6 +98,52 @@ const COMBINATION_MAP = {
 
 type CombinationType = keyof typeof COMBINATION_MAP;
 
+const SUPPORTED_LOCALES = ['en', 'de', 'fr'] as const;
+type Locale = (typeof SUPPORTED_LOCALES)[number];
```

Comment:
I'm not sure if typing this is much of benefit as you can't provide a custom translation (e.g. spanish with 'es') without changing the source code. So I'd suggest to remove these typings:

```suggestion
```